module/zfs/vdev_label.c

```diff
@@ -208,6 +208,40 @@ vdev_label_write(zio_t *zio, vdev_t *vd, int l, abd_t *buf, uint64_t offset,
 	    ZIO_PRIORITY_SYNC_WRITE, flags, B_TRUE));
 }
 
+void
+vdev_mmp_read(zio_t *zio, vdev_t *vd, int l, int n, abd_t *buf,
+    zio_done_func_t *done, void *private, int flags)
+{
+	uint64_t offset;
+	uint64_t size = VDEV_UBERBLOCK_SIZE(vd);
+
+	ASSERT(l >= 0);
+	ASSERT(l < VDEV_LABELS);
+	ASSERT(n >= 0);
+	ASSERT(n < MMP_BLOCKS_PER_LABEL);
+
+	offset = VDEV_UBERBLOCK_OFFSET(vd, VDEV_FIRST_MMP_BLOCK(vd) + n);
+
+	vdev_label_read(zio, vd, l, buf, offset, size, done, private, flags);
+}
+
+void
+vdev_mmp_write(zio_t *zio, vdev_t *vd, int l, int n, abd_t *buf,
+    zio_done_func_t *done, void *private, int flags)
```

Comment:
My feeling is it would more understandable to remove this `vdev_mmp_write` wrapper too and directly call `vdev_label_write` with the correct arguments.